README.md

```diff
@@ -15,136 +15,71 @@ When you have set up your repo
-│   └── workflows          # where you define your github actions
-│       └── moxunit.yml    # a yaml file that defines a github action
-├── lib                    # where you put the code from external libraries (mathworks website or other github repositories)
-│   └── README.md
-├── src                  # WHERE YOU PUT YOUR CODE
-│   ├── README.md
-│   └── miss_hit.cfg
-├── docs                 # where you write your documentation using sphinx
-│   ├── make.bat
-│   ├── Makefile
-│   ├── README.md
-│   ├── requirements.txt # to set up the python environment for the doc
-│   └── source           # the actual reStructured text file for your doc
-│       ├── conf.py
-│       ├── function_description.rst
-│       ├── index.rst
-│       ├── _static
-│       └── _templates
-├── tests               # where you put your unit tests
-|   ├── README.md
-|   └── miss_hit.cfg
-├── .travis.yml         # where you define the continuous integration done by Travis
-├── LICENSE
-├── README.md
-├── miss_hit.cfg        # configuration file for the matlab miss hit linter
-└── initEnv.m           # a .m file to set up your project (adds the right folder to the path)
-```
-
-## Keeping your code stylish: miss hit linter
+- anatomical defacing [ W I P ]
 
-### Matlab code style guide and quality
+- gzip the nii files in a BIDS raw folder
 
-We use the [MISS_HIT linter](https://github.com/florianschanda/miss_hit/) to
-automatically enforce / fix some code style issues and check for code quality.
+The core functions are in the `src` folder. This set of functions is intended to be added manually
+as a submodule to the analysis repo forked from
+[template_fMRI_analysis](https://github.com/cpp-lln-lab/template_fMRI_analysis)
 
-The linter is a python package that can be installed with:
+## How to install and use this template
 
-```bash
-pip3 install --user miss_hit
-```
+### Add as a submodule
 
-The rules followed by MISS_HIT are in the
-[MISS_HIT configuration file](./miss_hit.cfg).
+1. Open the terminal and navigate to your analysis folder forked from
+[template_fMRI_analysis](https://github.com/cpp-lln-lab/template_fMRI_analysis) and cloned locally
 
-To check the code style of the whole repository, you can can simply type:
+2. Type
 
 ```bash
-mh_style .
-```
+# navigate to the `lib` folder
+cd lib
 
-Some styling issues can be automatically fixed by using the `--fix` flag. You
-might need to rerun this command several times if there are a lot of issues.
+# add the submodule
+git submodule add https://github.com/cpp-lln-lab/CPP_sourceMRI
 
-```bash
-mh_style . --fix
+# initialize the nested submodule(s)
+git submodule update --init --recursive
 ```
 
-Code quality can be checked with:
+3. commit the new changes (the new submodule(s))
 
-```bash
-mh_metric .
-```
+### Dependencies
 
-To see only the issues that "break" the code quality rules set in the
-configuration file, type:
+Make sure that the following toolboxes are installed and added to the matlab
```

Comment:
Minor detail that must affect many of our docs: we list matlab as a matlab dependency that must be added to the path

Reply:
true, let me delete the "add to the path" part cause actually we never mind about that and it is also probably not recommended (versions/duplicates issues)

Reply:
```suggestion
Make sure that the following toolboxes are installed
```